multy_core/src/bitcoin/bitcoin_key.cpp

```diff
@@ -52,10 +54,20 @@ BitcoinPublicKey::~BitcoinPublicKey()
 std::string BitcoinPublicKey::to_string() const
 {
     CharPtr out_str;
-    THROW_IF_WALLY_ERROR(
-            wally_base58_from_bytes(
+    if (m_account_type == BITCOIN_ACCOUNT_P2PKH)
```

Comment:
It looks like both blocks of 'If' and 'else' are identical, could you please explain why?

Reply:
If both blocks should be identical, then there is no need to store `m_account_type`.